user-guide/Advanced_Functionality/Databases/Elasticsearch_database/Manually_Connecting_DMA_to_Elasticsearch_Cluster.md

```diff
@@ -0,0 +1,49 @@
+---
+uid: Manually_Connecting_DMA_to_Elasticsearch_Cluster
+---
+# Manually connecting a DMA to an Elasticsearch cluster
+
+You can connect a DataMiner Agent to an existing Elasticsearch cluster by manually adding a *DataBase* tag to the *DB.xml* file, as detailed below.
+
+> [!IMPORTANT]
+> From DataMiner 10.3.0/10.3.3 onwards, for a Cassandra cluster database, we recommend connecting your DMA to an Elasticsearch cluster in DataMiner Cube instead of in the *DB.xml* file. For more information, see [Configuring the general database settings in Cube](xref:Configuring_the_database_settings_in_Cube#cassandra-database).
+> For a Cassandra database per DMA, follow the steps below.
+
```

Comment:
@SLC-Laurens-Vercruysse Does this note suffice to make users aware of the possibility to use Cube from 10.3.3 onwards?

Reply:
This would suffice for now :)